mobile/src/modules/settings/screens/SettingsScreen.tsx

```diff
@@ -48,16 +67,44 @@ class Settings extends React.PureComponent<Props> {
     this.toggleConfirmDeletePrompt();
   }
 
+  handleLanguageChange = () => {
+  }
+
+  getLocales = () => {
+    return Object.keys(translations).map((locale) => {
+      return {
+        value: locale,
+        label: locale
+      };
+    });
+  }
+
   render () {
     return (
       <Screen>
         <FullHeightView>
-          <View style={styles.hintView}>
-            <Text theme={TextThemes.HEADER}>
-              {i18n.t(settings.screens.settings.title)}
-            </Text>
-          </View>
-          <View>
+          <View style={styles.container}>
+            {/* <BSelect
+                  // @ts-ignore bad .d.ts
+                  value={currentLocale.locale}
+                  items={this.getLocales()}
+                  onChange={this.handleLanguageChange}
+                  title={i18n.t(settings.screens.settings.changeLanguage)}
+                  placeholder={i18n.t(settings.screens.settings.changeLanguage)}
+            /> */}
```

Comment:
This is for later, not sure if we will end up needing it since i learned the app auto-detects language.